pkg/apis/batch/v2alpha1/defaults.go

```diff
@@ -46,4 +46,8 @@ func SetDefaults_ScheduledJob(obj *ScheduledJob) {
 	if obj.Spec.ConcurrencyPolicy == "" {
 		obj.Spec.ConcurrencyPolicy = AllowConcurrent
 	}
+	if obj.Spec.Suspend == nil {
+		obj.Spec.Suspend = new(bool)
+		*obj.Spec.Suspend = false
```

Comment:
We don't actually need this line since it's defaulted to false in L50 (or did you just want to be more explicit here?)